concepticondata/conceptlists/Nagano-2013-1256.tsv

```diff
@@ -0,0 +1,1257 @@
+Nagano-2013-1256-1104	1104	1977	all	98	ALL	
+Nagano-2013-1256-1105	1105	1978	all	98	ALL	
+Nagano-2013-1256-1106	1106	1979	completely			
+Nagano-2013-1256-1107	1107	1981	half	816	HALF	
+Nagano-2013-1256-1108	1108	1983	one third			
+Nagano-2013-1256-1109	1109	1985	a quarter			
+Nagano-2013-1256-1110	1110	1987	both			
+Nagano-2013-1256-1111	1111	1989	measure (v)	1506	MEASURE	
+Nagano-2013-1256-1112	1112	1990	scale			
+Nagano-2013-1256-1113	1113	1991	scale			
+Nagano-2013-1256-1114	1114	1993	heavy	1210	HEAVY	
+Nagano-2013-1256-1115	1115	1995	light	1052	LIGHT (WEIGHT)	
+Nagano-2013-1256-1116	1116	1997	many, much	1198	MANY	
+Nagano-2013-1256-1117	1117	1999	few, little	2924	A LITTLE	
+Nagano-2013-1256-1118	1118	2001	increase (v)	1041	INCREASE	
+Nagano-2013-1256-1119	1119	2003	decrease (v)	2864	DECREASE	
+Nagano-2013-1256-1120	1120	2006	I, me	1209	I	
+Nagano-2013-1256-1121	1121	2008	we, us [plural]	1212	WE	
+Nagano-2013-1256-1122	1122	2009	we, us [dual]	2634	WE TWO	
+Nagano-2013-1256-1123	1123	2011	you (sg.)	1215	THOU	
+Nagano-2013-1256-1124	1124	2013	you [plural]	1213	YOU	
+Nagano-2013-1256-1125	1125	2014	you two [dual]	2111	YOU TWO	
+Nagano-2013-1256-1126	1126	2016	he, him	1211	HE	
+Nagano-2013-1256-1127	1127	2018	she, her	1834	SHE	
+Nagano-2013-1256-1128	1128	2020	they, them [plural]	817	THEY	
+Nagano-2013-1256-1129	1129	2021	they two [dual]	2635	THEY TWO	
+Nagano-2013-1256-1130	1130	2023	oneself			
+Nagano-2013-1256-1131	1131	2025	this	1214	THIS	
+Nagano-2013-1256-1132	1132	2027	that	78	THAT	
+Nagano-2013-1256-1133	1133	2029	this	1214	THIS	
+Nagano-2013-1256-1134	1134	2031	that, that... over there			
+Nagano-2013-1256-1135	1135	2033	that, the			
+Nagano-2013-1256-1136	1136	2034	like, in this way, this kind of			
+Nagano-2013-1256-1137	1137	2035	that way, like that			
+Nagano-2013-1256-1138	1138	2037	that way, like that			
+Nagano-2013-1256-1139	1139	2039	here	136	HERE	
+Nagano-2013-1256-1140	1140	2041	there	1937	THERE	
+Nagano-2013-1256-1141	1141	2043	there	1937	THERE	
+Nagano-2013-1256-1142	1142	2045	this way, here			
+Nagano-2013-1256-1143	1143	2047	that way/that way (north)/that way (east)/that way (south)			
+Nagano-2013-1256-1144	1144	2049	that way, there			
+Nagano-2013-1256-1145	1145	2051	who	1235	WHO	
+Nagano-2013-1256-1146	1146	2053	what	1236	WHAT	
+Nagano-2013-1256-1147	1147	2055	which	727	WHICH	
+Nagano-2013-1256-1148	1148	2057	how	1239	HOW	
+Nagano-2013-1256-1149	1149	2059	where	1237	WHERE	
+Nagano-2013-1256-1150	1150	2061	where	1237	WHERE	
+Nagano-2013-1256-1151	1151	2063	why	1270	WHY	
+Nagano-2013-1256-1152	1152	2065	when	1238	WHEN	
+Nagano-2013-1256-1153	1153	2067	some day			
+Nagano-2013-1256-1154	1154	2069	how many?	822	HOW MANY	
+Nagano-2013-1256-1155	1155	2070	how many?	822	HOW MANY	
+Nagano-2013-1256-1156	1156	2071	how long?	2338	HOW LONG	
+Nagano-2013-1256-1157	1157	2072	how big?			
+Nagano-2013-1256-1158	1158	2074	something			
+Nagano-2013-1256-1159	1159	2076	some	1241	SOME	
+Nagano-2013-1256-1160	1160	2079	very much, very			
```

Comment:
Maybe 1311 VERY?

Reply:
More or less. It is supposedly only for amounts, and our 1311 VERY would be for "degree", but maybe I am being too picky. I will add it.